pages/common/qc.md

```diff
@@ -0,0 +1,25 @@
+# qc
+
+> CLI snippet manager to execute command snippets stored in notes in QOwnNotes.
```

Comment:
```suggestion
> Manage and execute command snippets stored in QOwnNotes notes.
```
What do you think of simplifying the description a littpe bit?

Reply:
LGTM, thank you!